boards/arc/qemu/qemu.dtsi

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
@galak: What's the recommendation here: dtsi file in the board directory, or in dts? I believe I've seen both.